packages/networkaccess/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@metamask/test-snap-networkaccess",
+  "version": "5.3.0",
+  "description": "MetaMask network-access Test Snap",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/test-snaps.git"
+  },
+  "publishConfig": {
+    "access": "public",
+    "registry": "https://registry.npmjs.org/"
+  },
+  "license": "ISC",
+  "files": [
+    "dist/",
+    "images/icon.svg",
```

Comment:
This icon is missing in directory and will not be displayed in Snaps list inside Settings tab/page. You can add it in `images` folder within `networkaccess` and it should work.

![Screenshot 2023-05-11 at 17 25 34](https://github.com/MetaMask/test-snaps/assets/13301024/5b69a34a-dd6f-4bbf-b7fb-0511bf5b32b5)

Reply:
resolved in latest commit